editorial-communication/cover-letter-submission.Rmd

```diff
@@ -11,24 +11,24 @@ output:
 ```{r include=FALSE}
 knitr::opts_chunk$set(fig.retina = 2)
 ```
+
 Dear Editor,
 
 We would like to submit our manuscript entitled “Not just for programmers: A friendly guide on the benefits of GitHub for accelerating collaborative research in Ecology and Evolution” for consideration to be published in *Nature Ecology & Evolution*. 
```

Comment:
Note change in title "Not just for programmers: How GitHub can accelerate collaborative and reproducible research in ecology and evolution"

Reply:
Thanks for catching that. I updated this PR with our new title